python_modules/dagster/dagster/_core/definitions/asset_sensor_definition.py

```diff
@@ -91,7 +121,29 @@ def _fn(context):
                     return
 
                 event_record = event_records[0]
-                result = materialization_fn(context, event_record.event_log_entry)
+
+                (
+                    context_param_name,
+                    event_log_entry_param_name,
+                ) = get_context_and_event_log_entry_param_names(materialization_fn)
+
+                resource_args_populated = validate_and_get_resource_dict(
+                    context.resources, name, resource_arg_names
+                )
+
+                # Build asset sensor function args, which can include any subset of
+                # context arg, event log entry arg, and any resource args
+                args = {
+                    **resource_args_populated,
+                    **({context_param_name: context} if context_param_name else {}),
```

Comment:
matter of taste (so feel free to ignore), but I think this might be easier to read as:

```
args = resource_args_populated
if context_param_name:
    args[context_param_name] = context
if event_log_entry_param_name:
    args[event_log_entry_param_name] = event_record.event_log_entry
```